eslint.config.js

```diff
@@ -0,0 +1,61 @@
+const js = require('@eslint/js');
+const globals = require('globals');
+
+
+const styleRules = {
+    quotes: ['error', 'single', { avoidEscape: true }],
+    'no-trailing-spaces': 'error',
+    indent: 'error',
+    'linebreak-style': ['error', 'unix'],
+    semi: ['error', 'always'],
+    'brace-style': ['error', '1tbs', { allowSingleLine: true }],
+    'keyword-spacing': 'error',
+    'space-before-blocks': 'error',
+    'space-before-function-paren': [
+        'error',
+        { anonymous: 'always', named: 'never' },
+    ],
+    'no-mixed-spaces-and-tabs': 'error',
+    'comma-spacing': ['error', { before: false, after: true }],
+    'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+};
+
+
+module.exports = [
```

Comment:
Can we add a global ignore for `example/public/**`? If you happen to have ran the example app it throws loads of lint errors on the built `application.js` without that.

Reply:
The [global ignore](https://eslint.org/docs/latest/use/configure/configuration-files#globally-ignoring-files-with-ignores) for this hasn't gone in the right place